test/gui/tst_syncZipFile/test.feature

```diff
@@ -0,0 +1,25 @@
+Feature: .zip files sync
+
+    As a user
+    I want to be able to upload zip file into the sync root
+    So that extracted files and folders are available on the server
```

Comment:
I would suggest to not add new folder for this one scenario. Maybe you can add it to already existing feature file